sematic/config/config.py

```diff
@@ -10,6 +10,31 @@
 from sematic.config.config_dir import get_config_dir
 from sematic.config.settings import MissingSettingsError
 from sematic.config.user_settings import UserSettingsVar, get_user_setting
+from sematic.versions import version_as_string
+
+logger = logging.getLogger(__name__)
+
+# Support for dropping columns added in 3.35.0
+MIN_SQLITE_VERSION = (3, 35, 0)
+
+
+def _check_sqlite_version():
+    # Standard Library
+    import sqlite3
```

Comment:
Why do this import down here? It's a part of the stdlib, so there's no chance of the import failing above

Reply:
This was a micro-optimization to not import potentially expensive modules. Moved.